src/routes/index.ts

```diff
@@ -118,6 +122,20 @@ router.get(config.MANAGING_OFFICER_CORPORATE_URL + config.REMOVE + config.ID, au
 router.get(config.TRUST_INFO_URL, authentication, navigation.hasBOsOrMOs, trustInformation.get);
 router.post(config.TRUST_INFO_URL, authentication, navigation.hasBOsOrMOs, ...validator.trustInformation, checkTrustValidations, trustInformation.post);
 
+if (isActiveFeature(FEATURE_FLAG_ENABLE_TRUSTS_WEB)) {
+  router.get(
+    config.TRUST_DETAILS_URL,
+    authentication,
+    trustDetailsController.get,
+  );
+
+  router.post(
+    config.TRUST_DETAILS_URL,
+    authentication,
```

Comment:
You need to add validation checks middleware if you want to catch errors

Reply:
it is other story

Reply:
Yup, what Dmitry said - This has not being forgotten and is being dealt with in a future story.